web2/htdocs/index.html

```diff
@@ -1097,9 +1097,8 @@
                   </style>
                 </defs>
 
-                [[
-                    var total = Math.max(_.store.used, _.store.threshold, _.store.projected, _.store.total);
-                ]]
+                [[# Base value of 1byte, add 20% padding for spacing ]]
+                [[ var total = Math.max(_.store.used, _.store.threshold, 1) * 1.2; ]]
```

Comment:
Probably ought to bump the minimum to 1MB, since 1b will present as "definitely something wrong with SHIELD" not just "nothing using this and no threshold"